.github/workflows/ci.yml

```
@@ -20,36 +23,27 @@ jobs:
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
-        nxf_ver: ['19.10.0', '']
+        nxf_ver: ["21.04.0", ""]
```

```suggestion
        nxf_ver: ["21.04.2", ""]
```

Could go for the latest stable here - just do a `nf-core bump-versions -n "21.04.2"` once and should be good to go 👍🏻

---

Resolved in #d61cc05ef8c7b4387de3bf4f241a7f678e1304bf